src/components/buttons/Pin/index.js

```diff
@@ -17,6 +17,7 @@ const Wrapper = styled.button`
     border: 1px solid ${colors.DIVIDER};
     background: ${colors.WHITE};
     transition: all 0.3s;
+    font-family: Roboto, sans-serif;
```

Comment:
Are we sure we want to force a specific font for some components?

Reply:
Fixed by removing the 'dot' letter and used css :before